install/upgrade/versions/1.6.0.sh

```diff
@@ -67,3 +67,14 @@ fi
 if [[ ! -d $HESTIA/data/api ]]; then
     cp -rf $HESTIA_INSTALL_DIR/api $HESTIA/data/
 fi
+
+# Update Cloudflare address
+if [ -f /etc/nginx/nginx.conf ] && [ "$(grep 'set_real_ip_from 2405:8100::/32' /etc/nginx/nginx.conf)" = "" ];then
```

Comment:
You also need to update ip4 ip addresses

Reply:
After sorting, there are actually no change in IPv4